inksdk/inksample/src/main/java/com/microsoft/device/inksample/MainActivity.kt

```diff
@@ -0,0 +1,124 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License.
+ */
+
+package com.microsoft.device.inksample
+
+import android.graphics.Color
+import android.graphics.Paint
+import android.os.Build
+import android.os.Bundle
+import android.util.TypedValue
+import android.view.View
+import android.webkit.WebView
+import android.webkit.WebViewClient
+import android.widget.ImageView
+import android.widget.Switch
+import androidx.annotation.RequiresApi
+import androidx.appcompat.app.AppCompatActivity
+import androidx.core.graphics.blue
+import androidx.core.graphics.green
+import androidx.core.graphics.red
+import com.microsoft.device.ink.InkView
+import com.microsoft.device.ink.InkView.DynamicPaintHandler
+import com.microsoft.device.ink.InputManager
+
+class MainActivity : AppCompatActivity() {
+
+    private lateinit var inkView: InkView
+    private lateinit var webView: WebView
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_main)
+        inkView = findViewById(R.id.inkView)
+        webView = findViewById<WebView>(R.id.webView)
+        webView.webViewClient = WebViewClient()
+        webView.loadUrl("https://en.wikipedia.org/wiki/Special:Random")
+        val webSettings = webView.settings
+        webSettings.javaScriptEnabled = true
+    }
+
+    @RequiresApi(Build.VERSION_CODES.Q)
```

Comment:
Doest it makes sense to change the `minSdk` directly to 29 (q)? (not sure if the ink sdk can be used with lower apis than 29)